pywbem/exceptions.py

```diff
@@ -222,6 +222,9 @@ class ParseError(Error):
 
     Derived from :exc:`~pywbem.Error`.
 
+    Optionally, the CIM-XML request and response data are attached to the
```

Comment:
The work optionally bothers me because it does not define the condition that is optional.

Reply:
Actually, the optionally refers to the ability of the exception class. In actual usage, I think we can guarantee that both are attached.

Reply:
So, since this is the class description, so we want to describe the abilities of the class, or how we use it?

Reply:
I think we just want to state what the characteristics of the optionality are. i.e. generally wney attached and when not attached, even if it just points to behavior in the exception class.  It is just that the word optionally only means you cannot depend on something unless the condition for the optionality is understood.

Reply:
Ok, I will add something to clarify that.